CONTRIBUTING.md

```diff
@@ -1,39 +1,65 @@
+To be on the list, project repositories should adhere to these quality standards 
+(https://goreportcard.com/report/github.com/ **github_user** / **github_repo**):
 
-- Code functions as documented and expected
-- Generally useful to the wider community of Go programmers
+- Code functions as documented and expected;
+- Generally useful to the wider community of Go programmer
 - Actively maintained
   - Regular, recent commits
-  - Or, for finished projects, issues and pull requests are responded to
+  - Or, for finished projects, issues and pull requests are responded to generally within 2 weeks
 - Stable or progressing toward stable
 - Thoroughly documented (README, pkg.go.dev doc comments, etc.) in english language, so everyone is able to understand the project's intention and how it works
 - Tests, where practical. If the library/program is testable, then coverage should be >= 80% for non-data-related packages and >=90% for data related packages. **Notice**: the tests will be reviewed too. We will check your coverage manually if your package's coverage is just a benchmark results.
+- The project has a Go Report Card value of A.
+- The project must have at least one official version-numbered release that allows go.mod files
+  to list the file by version number. It should be of the form vX.X.X.
+
+
+## Preparing for review
+
+To make it easy to review, we ask that you do the following to the README file in your proect:
+- Add a badge that points to the matching pkg.go.dev link. You can use https://pkg.go.dev/badge/ to create the badge;
+- Add a badge that points to the matching Go Report Card report. To generate the report, go to https://goreportcard.com/. Click on the report badge in the upper right corner to see details on how to add the badge to your Readme. The report card must correspond to the latest official release of the project or the current development tip;
+- Add a badge that points to a matching coverage report. There are many free ones you can choose from, including codecov, coveralls, and gocover. Also acceptable is generating your own badge as part of a continuous integration process. The report card must correspond to the latest official release of the project or the current development tip;
+
+You will be asked to provide these links when submitting.
+
+## How to add an item to the list
+
+Open a pull request against the README.md document that adds the repository to the list.
+
+- The pull request should add one and only one item to the list;
+- The added item should be in alphabetical order within its category;
+- The link should be the name of the package or project;
+- Descriptions should be clear, concise, and non-promotional;
+- Descriptions should follow the link, on the same line and end with a punctuation mark;
+- Remember to put a period `.` at end of the project description;
+- Links to projects that are version 2 or higher should end in /vX/, where X is the major version number;
+
```

Comment:
I think an additional rule can be added here, "Add a link to the source repository instead of linking to a dedicated website when both exist."
What do you think @avelino?